### PR TITLE
Make React a peerDep, allow 0.14

### PR DESCRIPTION
**Pull request by andreypopp**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
     "node": ">=0.10.x"
   },
   "main": "component/index.js",
-  "dependencies": {
-    "react": "^0.13.1"
+  "peerDependencies": {
+    "react": ">= 0.13.1, < 0.15.0"
```
Comment (Theadd) [inline package.json]: `"react": ">= 0.13.1, < 0.15.0"` does NOT match with `0.14.0-rc1`.

It should be something similar to `"react": ">=0.13.1 || ^0.14.0-rc1"`

Comment (andreypopp) [inline package.json]: Yeah, actually `>= 0.13.1 < 0.14 || ^0.14.0-rc1` so we disallow `0.15`.

Comment (andreypopp) [inline package.json]: Fixed.

**Comment by jessy1092** [comment]
@andreypopp  Thanks for the PR. But I would like to wait until `react: 0.14` release. :)

**Comment by tomchentw** [comment]
@jessy1092 it's time for this! XD

**Comment by jessy1092** [comment]
:+1:

**Comment by tomchentw** [comment]
:+1:

**Comment by tomchentw** [comment]
Will you release a new version?

**Comment by tomchentw** [comment]
@jessy1092 I found that `^0.14.0-rc1` can't match `^0.14.0` LOL

**Comment by jessy1092** [comment]
@tomchentw lol, I would modify to original version. Just for 0.14.0

```
"peerDependencies": {
   "react": ">= 0.13.1 < 0.15.0"
},
```

Thanks everyone.

**Comment by tomchentw** [comment]
Thanks @jessy1092 . Please also release a new version then.

**Comment by tomchentw** [comment]
Thanks for releasing `v0.4.2` for this!

**Pull request merged**